Pico map/Assets/Scripts/quit.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```

Comment:
What does this script do? Every script should have some header comments telling us what it is doing.

Reply:
yes this dose it exits the build

Reply:
I'm not specifically asking what this script does, just that every script needs comments detailing what it does.